tests/difference/core/model/src/model.ts

```diff
@@ -458,36 +465,25 @@ class CCVProvider {
   };
 
   onReceiveSlash = (data: Slash) => {
-    let infractionHeight = undefined;
-
-    if (data.vscID === 0) {
-      infractionHeight = this.initialHeight;
-    } else {
-      infractionHeight = this.vscIDtoH[data.vscID];
-    }
 
+    // Check validator status
     if (this.m.staking.status[data.val] === Status.UNBONDED) {
       this.m.events.push(Event.RECEIVE_SLASH_REQUEST_UNBONDED);
       return;
     }
 
-    if (data.isDowntime) {
-      this.m.events.push(Event.RECEIVE_DOWNTIME_SLASH_REQUEST);
-    } else {
-      this.m.events.push(Event.RECEIVE_DOUBLE_SIGN_SLASH_REQUEST);
-    }
+    this.m.events.push(Event.RECEIVE_DOWNTIME_SLASH_REQUEST);
```

Comment:
Related to above would double-sign packets get to this control flow? It's a little unclear to me

Reply:
Nop, double-sign packets are dropped before in the [onReceive](https://github.com/cosmos/interchain-security/blob/0cb44b5942ac4a574ede045daf61a31bdf06b59d/tests/difference/core/model/src/model.ts#L424) function so they aren't added to the queue and processed here in `onReceiveSlash`.